requirements/dist-build-constraints.in

```diff
@@ -5,7 +5,6 @@
 setuptools >= 34.4
 
 # Plugins
-setuptools_scm[toml] >= 3.5
 setuptools_scm_git_archive >= 1.1
```

Comment:
This must be in sync with `pyproject.toml` at all times, as the comment at the top instructs — this dependency is not needed anymore since it should've been removed from `pyproject.toml`.